styles.css

```diff
@@ -144,6 +142,89 @@ h1 {
   width: 10.635416984558105rem;
 }
 
+#hamburger-menu {
+  background-image: url(images/Overlay.png);
+
+  /* height: 37.75rem;
+  width: 23.4375rem; */
+  flex-direction: column;
+  justify-content: center;
+  padding: 7.125rem 1.5rem;
+  width: 375px;
+  height: 750px;
+  display: none;
+  position: absolute;
+  opacity: 0.8;
+  top: 0;
+  padding-top: 44px;
+}
+
+.blur {
+  filter: blur(5px);
+  opacity: 0.8;
+}
```

Comment:
- You have done great so far but there are some changes you need to make before moving to the next milestone. :+1: :100: 
- [ ] As stated in the project requirement, you need to try to replicate the Figma design as much as you can, currently, I think some sections of your mobile menu need a bit of adjustment. The Hamburger menu is currently visible on desktop screens, I'll suggest you fix this as its meant to be visible only on mobile. :+1: :100: 
- - Screenshot:
![Selection_093](https://github.com/umarahgul/Portfolio/assets/107632780/e4d4f652-a9df-46f6-8d7f-842f53caa3ee)

- [ ] Also, currently the menu item on the desktop screen doesn't take you to the specific part of the page when a user clicks on it, I'll suggest you implement this also as it helps the user navigate the website easily :+1: :100: